<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenuWidget</class>
 <widget class="QWidget" name="UserMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#17212B;
color: rgb(86, 115, 129);
font-size: 17px;
font-weight:bold;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <widget class="QPushButton" name="UserHeaderIcon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>63</width>
        <height>63</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#UserHeaderIcon{
border-color: rgb(255, 85, 0);
border-width: 10px;
border-radius: 30%;
background-color:
rgb(0, 0, 0)
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="UserName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">text-align:left;
padding-left:20px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="6" column="0">
       <widget class="QPushButton" name="settingButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/settings.png</normaloff>:/resources/settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="callButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>通话</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/callButton.png</normaloff>:/resources/callButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="contactButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>联系人</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/contact-book.png</normaloff>:/resources/contact-book.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="collectionButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>我的收藏</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/collection.png</normaloff>:/resources/collection.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>68</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
padding-left:0px;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:17px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:400;&quot;&gt;Chat Client Gui&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="createGroupButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>创建群组</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/createGroup.png</normaloff>:/resources/createGroup.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="nightButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>夜间模式</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/night.png</normaloff>:/resources/night.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="createChannelButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #17212B;
color:#E9E8E8;
border:none;
}
QPushButton:hover {
background-color: #242F3D;
border:none;
}</string>
        </property>
        <property name="text">
         <string>创建频道</string>
        </property>
        <property name="icon">
         <iconset resource="../res/res.qrc">
          <normaloff>:/resources/customerButton.png</normaloff>:/resources/customerButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
